### DISCUSSION TITLE
Typedef for useForm in v7.40.0

### DISCUSSION THREAD

**Discussion by ewylie25** [react-hook-form/react-hook-form#9630] [General] [answered]
Are updated type definitions missing in v7.40.0 for using the `values` prop on `useForm`? Or am I misunderstanding the release notes/updated docs

I noticed they don't seem to be here:
![image](https://user-images.githubusercontent.com/2636781/208536350-9af7ffc0-8eb8-49e5-a3db-7655435269fe.png)

and I'm getting this error trying to utilize in my project:
```
Argument of type '{ mode: "onSubmit"; values: IMyModel; }' is not assignable to parameter of type 'Partial<{ mode: keyof ValidationMode; reValidateMode: "onBlur" | "onChange" | "onSubmit"; defaultValues: { foreignTaxRedeterminationKey?: number | undefined; enterpriseKey?: number | undefined; ... 25 more ...; contestedChangeInTax?: number | ... 1 more ... | undefined; }; ... 6 more ...; delayError: number; }>'.
  Object literal may only specify known properties, and 'values' does not exist in type 'Partial<{ mode: keyof ValidationMode; reValidateMode: "onBlur" | "onChange" | "onSubmit"; defaultValues: { foreignTaxRedeterminationKey?: number | undefined; enterpriseKey?: number | undefined; ... 25 more ...; contestedChangeInTax?: number | ... 1 more ... | undefined; }; ... 6 more ...; delayError: number; }>'
```

**Comment by Moshyfawn** [comment, marked as answer]
`values` is a new feature of [7.41.0](https://github.com/react-hook-form/react-hook-form/releases/tag/v7.41.0)

**Reply by ewylie25** [reply]
Got it - I assumed because it was in some of the 7.40.0-next* releases that it was rolled into 7.40.0. Thank you! Grabbing 7.41.0 resolved it

**Reply by Moshyfawn** [reply]
It got pushed back a little, yea. Glad I could help ;)
